[DOC-2299] Removed custom secret manager feature flag (#1739)
harness/developer-hub: docs/first-gen/firstgen-platform/techref-category/api/use-custom-secrets-manager-api.md

helpdocs_is_published: true
---

> Harness provides first-class support and built-in integration for the most common secret managers. If you are using a [secret manager](../../security/secrets-management/secret-management.md) that Harness does provide first-class support for, you can configure and use your secret manager by using the Harness Custom Secrets Manager.

Reviewer: ```suggestion
Harness provides first-class support and built-in integration for the most common [secret managers](../../security/secrets-management/secret-management.md). If you're not using a secret manager that's supported, you can configure it by using the Harness Custom Secrets Manager.
```
